### DISCUSSION TITLE
Fooocus colab issue for changing model

### DISCUSSION THREAD

**Discussion by jojo97126** [lllyasviel/Fooocus#3396] [Q&A] [answered]
Hi, i'm very new on fooocus and i discovered colab but i want to change the base model for another one. When i try to generate image with a new model that i downloaded it keep saying this in the command :

[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
^C

and on the UI it stay blocked on "loading models..." then get error

this is how i wrote it : ![image_2024-07-28_154230027](https://github.com/user-attachments/assets/5c561e92-8c14-47bb-8845-94cf9af230b2)

please help me

**Comment by mashb1t** [comment]
Colab (free) features are limited, but you can load the preset => modify the entrypoint script by adding e.g. `--preset realistic`

**Reply by jojo97126** [reply]
thanks for your reply, but when you say that, are you saying that i can't load new models into colab ?

**Reply by mashb1t** [reply]
no that 100% works, but you may have to start Fooocus there with a preset containing your model instead of loading it after starting, depending on the modifications you did there.

**Reply by jojo97126** [reply]
ok thanks for your reply, i tried to do a --preset anime, and i got some nice result (i wanna do anime characters btw) but when you say that i need to start Fooocus there with a preset containing my model what do you mean ?
![image_2024-07-28_165826241](https://github.com/user-attachments/assets/b734463d-f90d-4d89-9915-8fe60bcfd3dd)

**Reply by mashb1t** [reply, marked as answer]
you can copy an existing preset, exchange the model  with yours incl. LoRAs etc. and start Fooocus using your preset. But only if switching the model still doesn't work.

**Reply by jojo97126** [reply]
THANK YOU SOOOO MUCH :) 👍 
i keep a version non modified of the raw anime presset
have a good one !

**Reply by jojo97126** [reply]
now i got an issue when my image is going to finish the process it give me an error
![image](https://github.com/user-attachments/assets/18f9017c-1400-4552-ba7b-d74691f2770c)
the console :
![image](https://github.com/user-attachments/assets/bb4fb9c7-04e4-4a71-bf61-f4333bd91c9b)

**Reply by mashb1t** [reply]
see https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md#c-then-quit
Upgrade to Colab Pro or don't use specific features in Colab as described in the readme.

**Reply by jojo97126** [reply]
ok thank you so much
